src/Views/View.js

```diff
@@ -71,5 +73,5 @@ class Views {
             </div>`;
   }
 }
-export const view = new Views(sources, settings);
-view.initialContent();
+export const myView = new View(sources, settings);
+myView.initialContent();
```

Comment:
can we call initialContent inside of controller?

Reply:
I call it inside controller init() now.